api/src/controllers/login.js

```diff
@@ -17,8 +17,7 @@ const template = require('../services/template');
 
 const templates = {
   login: {
-    content: null,
-    file: 'index.html',
+    file: path.join(__dirname, '..', 'templates', 'login', 'index.html'),
```

Comment:
I decided to inline the whole file path so it calculates it at runtime not every time it's called.